pkgs/tools/system/bashtop/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "bashtop";
+  version = "0.9.25";
+
+  meta = with lib; {
+    homepage = "https://github.com/aristocratos/bashtop";
+    license = licenses.asl20;
+    platforms = platforms.linux;
+    maintainers = [ maintainers.tretinha ];
+  };
+
+  installFlags = [ "PREFIX=${placeholder "out"}" ];
+
+  src = fetchFromGitHub {
```

Comment:
Swap where meta and src are, its a convention.

Structure:
- pname
- version
- src
- phases and extra settings about phases
- meta

Reply:
Hey, thank you for your review. I think I added a wrong commit here. I'll close it and start from a clean slate.